jsk_naoqi_robot/jsk_201504_miraikan/README.md

```diff
@@ -1,18 +1,34 @@
 jsk_201504_miraikan
 ===================
 
-how to run demo
---------------
+What is this?
+-------------
 
-For Japanese and English demo,
-Please confirm the language (Japanese/ English). You can change it from Pepper's tablet -> setup. (Note: This function is under construction.)
+Pepper@JSK introduces him/ herself.  
+Pepper can speak three languages (Japanese, Chinese and English).  
 
-For Chinese demo,
-please move mp3 files in file directory you want Pepper to speak under ``` /home/nao/audio_file . If not, please make ```audio_file``` folder in ```/home/nao/```. You can use ```scp <file> nao@<Pepper IP>:/home/nao/audio_file/```.
+How to run demo?
+----------------
+
```

Comment:
I have to add Naoqi version and ROS version which I confirmed that this works.
In addition, I have to add version of other packages  which this package depends on.